Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2003-10-04
Initial Package Version: 4.44
Origin: BLFS Book
Upstream Status: N/A
Description: Uses FHS compliant paths.

$LastChangedBy: djensen $
$Date: 2005-06-17 11:23:19 -0500 (Fri, 17 Jun 2005) $

--- pine4.58/doc/pine.1.orig	2003-12-06 01:16:10.000000000 +0000
+++ pine4.58/doc/pine.1	2003-12-06 01:17:59.000000000 +0000
@@ -333,11 +333,11 @@
 .br
 /etc/mime.types	System-wide file ext. to MIME type mapping
 .br
-/usr/local/lib/pine.info	Local pointer to system administrator.
+/etc/pine.info	Local pointer to system administrator.
 .br
-/usr/local/lib/pine.conf	System-wide configuration file.
+/etc/pine.conf	System-wide configuration file.
 .br
-/usr/local/lib/pine.conf.fixed	 Non-overridable configuration file.
+/etc/pine.conf.fixed	 Non-overridable configuration file.
 .br
 /tmp/.\\usr\\spool\\mail\\xxxx	Per-folder mailbox lock files.
 .br
--- pine4.58/pine/osdep/os-lnx.h.orig	2003-12-06 01:26:39.000000000 +0000
+++ pine4.58/pine/osdep/os-lnx.h	2003-12-06 01:40:59.000000000 +0000
@@ -163,10 +163,10 @@
 
 /*----- System-wide config file ----------------------------------------*/
 #ifndef SYSTEM_PINERC
-#define SYSTEM_PINERC             "/usr/local/lib/pine.conf"
+#define SYSTEM_PINERC             "/etc/pine.conf"
 #endif
 #ifndef SYSTEM_PINERC_FIXED
-#define SYSTEM_PINERC_FIXED       "/usr/local/lib/pine.conf.fixed"
+#define SYSTEM_PINERC_FIXED       "/etc/pine.conf.fixed"
 #endif
 
 
